sdks/open-format/src/core/contract.ts

```diff
@@ -454,6 +454,12 @@ export async function deploy({
 
```

Comment:
I think we need another `invariant` line here, something like...

```tsx
invariant(nftStorageToken, 'Please set a NFT storage token, more information https://docs.openformat.simpleweb.co.uk/guides/deploying');
```

This way, it'll stop the `NFTStorage` below failing to be created with an empty token.

Reply:
> Refe

ah yep I get you.